ansible/roles/chroma/templates/env.j2

```diff
@@ -0,0 +1,18 @@
+{{ansible_managed}}
+
+KOALA_CLIENT_ID={{ secret_oauth2_proxy.client_id }}
+KOALA_CLIENT_SECRET={{ secret_oauth2_proxy.client_secret }}
+S3_ACCESS_KEY_ID={{ secret_chroma.s3_access_key }}
+S3_SECRET_ACCESS_KEY={{ secret_chroma.s3_secret_access_key }}
+S3_REGION={{ secret_chroma.s3_region }}
+S3_BUCKET_NAME={{ secret_chroma.s3_bucket }}
+S3_ENDPOINT_URL=http://s3-{{secret_chroma.s3_region}}.amazonaws.com
+HTTP_PORT=34971
+DB_URL="postgresql:///chroma"
+STORAGE_ENGINE="S3"
+SERVICE_TOKENS=""
```

Comment:
We should probably add a service token right from the start, so we can perform the migration